jwql/website/apps/jwql/static/js/jwql.js

```diff
@@ -75,53 +81,53 @@ function search(n_proposals) {
 
```

Comment:
In bad news, I am seeing that the search function isn't working correctly now. Typing in a particular number isn't accurately filtering out proposals that don't start with that number. Do you see this behavior, too? 

I'm not sure at the moment what might be causing it - maybe a misinterpretation of what's being typed into the search box, or a misassignment of the `proposal` attribute to each thumbnail div. If you don't have an immediate idea of why this might be happening, I'm happy to do a deeper dive.

Reply:
Though the sort functions are still working, interestingly...

Reply:
@laurenmarietta Hmmm, good catch, I'll look into this.